packages/ng/callout/src/lib/callout/callout.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Palette, SizeMS } from '@lucca-front/ng/core';
+
+@Component({
+	selector: 'lu-callout',
+	standalone: true,
+	imports: [CommonModule],
+	templateUrl: './callout.component.html',
+	styleUrls: ['./callout.component.scss'],
+})
+export class CalloutComponent {
+	@Input()
+	/**
+	 * The title of the callout
+	 */
+	title: string;
+
+	@Input()
+	/**
+	 * Which palette should be used for the entire callout.
+	 * Defaults to none (inherits parent palette)
+	 */
+	palette: Palette = 'none';
+
+	@Input()
+	/**
+	 * Which size should the callout be? Defaults to medium
+	 */
+	size: SizeMS = 'm';
```

Comment:
With the possibilities in the variable name, won't it be annoying in the future if we add more?

Reply:
I think that ideally, we'd have to define a list of size names and use them everywhere to have more consistency tbh.
This is more of an example one, but I agree that we should just use the same names everywhere and have types to define which ones we want to allow where (for instance, have a type that's `SizeSML` that is defined by `'small' | 'medium' | 'large'`, one `SizeSM` that's the same without large, etc.

I don't know if this naming change should be done starting now or not tho.